.github/workflows/e2e-advanced-tests.yml

```
@@ -0,0 +1,296 @@
+          - all
+          - network
+          - concurrent
+          - security
+          - accessibility
+
+jobs:
+  advanced-e2e-tests:
+    name: Advanced E2E Tests
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)